x-pack/plugins/security_solution/public/common/components/drag_and_drop/drag_classnames.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/** This class is added to the document body while dragging */
```

Comment:
oh this is going to be fun!!! I moved them in packages to be shared between security solution and timelines

Reply:
Yea... 😅